### DISCUSSION TITLE
cannot verification code

### DISCUSSION THREAD

**Discussion by Nileeyee** [community/community#136921] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

 I'm cannot verification code because email  disabled  google account.

**Comment by K3irill** [comment]
https://support.google.com/accounts/answer/40695?hl=en

**Comment by LiteBrite82** [comment]
Hi @Nileeyee ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
